src/axom/inlet/Container.hpp

```diff
@@ -1231,6 +1231,34 @@ class Container : public Verifiable<Container>
   template <typename Func>
   void forEachCollectionElement(Func&& func) const;
 
+  /*!
+   *****************************************************************************
+   * \brief Applies a provided function to nested elements of the calling table
+   * and stores the result in a range
```

Comment:
On my first pas through the docs, I am not sure what "...in a range" means at the end

Reply:
Update after seeing the rest of the docs -- the results are put into an output iterator ``\a output``